tests/CppUTestExt/MockExpectedCallTest.cpp

```diff
@@ -528,7 +528,7 @@ TEST(MockExpectedCallComposite, isOnObject)
 {
     composite.onObject(&composite);
     SimpleString info("(object address: ");
-    info += StringFromFormat("%p", &composite);
+    info += StringFromFormat("%p", static_cast<void*>(&composite));
```

Comment:
Could you change this to a C style cast? Within CppUTest we prefer not using the "new" style C++ casts.